assets/style.css

```diff
@@ -0,0 +1,998 @@
+  font-family: "Roboto Condensed";
+  src: url(fonts/RobotoCondensed-Regular.ttf);
+}
+
+html * {
+  font-family: "Roboto Condensed", arial, sans-serif;
+}
+
+body {
+  background-color: #f4f5f6;
+  min-width: 450px;
+}
+
+.container,
+.container-fluid {
+  max-width: 1310px;
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
+}
+
+.text-primary {
+  color: #f72210 !important;
+}
+
+.border *,
+.border {
+  border: solid 2px black !important;
+}
+
+.border-none {
+  border: none !important;
+}
+
+/* links */
+
+a:hover,
+a:link,
+a:active,
+a:visited {
+  color: currentcolor;
+  text-decoration: none;
+}
+
+.link-c-red a:hover,
+.link-c-red:hover {
+  color: #f72210;
+}
+
+.link-c-red2 a:hover,
+.link-c-red2:hover {
+  color: #f92859;
+}
+
+.link-c-black a:hover,
+.link-c-black:hover {
+  color: black !important;
+}
+
+.link-u-red a:hover,
+.link-u-red:hover {
+  text-decoration: underline red !important;
+}
+
+.link-b-gray a:hover,
+.link-b-gray:hover {
+  background-color: #dbdcdd;
+}
+
+/* main */
+
+main {
+  max-width: 1310px;
+}
+
+@media (min-width: 992px) {
+  .bg-lg-white {
+    background-color: white !important;
+  }
+}
+
+main .container-lg-fluid {
+  width: 100%;
+  padding-right: 15px !important;
+  padding-left: 15px !important;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+header {
+  position: fixed;
+  top: 0;
+  z-index: 1;
+}
+
+@media (min-width: 992px) {
+  header {
+    position: static;
+  }
+}
+
+/* top navbar */
+
+.logo > img {
+  width: 132px;
+  margin-left: 15px;
+}
+
+.temp-icon > img {
+  width: 24px;
+  height: 24px;
+}
+
+.subscribe-btn,
+.login-btn {
+  padding: 6px 2px;
+  font-weight: 700;
+  font-size: 14px;
+  margin-left: 10px;
+  line-height: 14px;
+}
+
+.toggler-icon {
+  width: 50px;
+  height: 50px;
+  padding-left: 11px;
+  padding-right: 11px;
+}
+
+.toggler-icon > span,
+.toggler-icon::before,
+.toggler-icon::after {
+  display: block;
+  width: 24px;
+  height: 4px;
+  background-color: #fb9a94;
+  margin: 2px;
+}
+
+.toggler-icon::before,
+.toggler-icon::after {
+  content: "";
+}
+
+@media (min-width: 481px) {
+  .subscribe-btn {
+    flex-basis: 121px;
+    margin-right: 6px;
+  }
+
+  .subscribe-btn,
+  .login-btn {
+    padding: 6px 8px 6px 9px;
+    font-weight: 700;
+    font-size: 18px;
+    margin-left: 10px;
+    line-height: 14px;
+  }
+
+  .login-btn {
+    padding-right: 2px;
+  }
+
+  .subscribe-btn::after {
+    width: 8px;
+    height: 14px;
+    content: "";
+    display: inline-block;
+    margin-left: 10px;
+    background: url(images/icon-arrow-right-red.svg) center / contain no-repeat;
+  }
+
+  .toggler-icon > span,
+  .toggler-icon::before,
+  .toggler-icon::after {
+    width: 30px;
+  }
+
+  .toggler-icon {
+    width: 74px;
+  }
+}
+
+@media (min-width: 576px) {
+  .temp-icon {
+    margin-left: 23px;
+    margin-right: 5px;
+  }
+
+  .temp-deg {
+    font-size: 15px;
+  }
+}
+
+@media (min-width: 641px) {
+  .temp-location {
+    display: block !important;
```

Comment:
you can use the `d-block` bootstrap class here

Reply:
I think it's not possible because there's no `641px` break point in bootstrap.